draft-irtf-cfrg-aegis-aead.md

```diff
@@ -937,11 +937,11 @@ The `Enc` function encrypts a 128-bit input block `xi` using the state `{S0, ...
 
 Inputs:
 
-- `xi`: the input block.
+- `xi`: the 128-bit input block.
```

Comment:
This matches AEGIS-128L, but there are other instances of the same thing below that don't at the moment.